ScanAnalysis/scan_analysis/mapping/map_Undulator.py

```diff
@@ -0,0 +1,25 @@
+from scan_analysis.base import AnalyzerInfo as Info
+
+from scan_analysis.analyzers.Undulator.MagSpecStitcherAnalysis import MagSpecStitcherAnalysis
+from scan_analysis.analyzers.Undulator.CameraImageAnalysis import CameraImageAnalysis
+from scan_analysis.analyzers.Undulator.VisaEBeamAnalysis import VisaEBeamAnalysis
+from scan_analysis.analyzers.Undulator.Rad2SpecAnalysis import Rad2SpecAnalysis
+from scan_analysis.analyzers.Undulator.HiResMagCamAnalysis import HiResMagCamAnalysis
+
+undulator_analyzers = [
+    Info(analyzer_class=MagSpecStitcherAnalysis,
+         requirements={'AND': ['U_BCaveICT', 'U_BCaveMagSpec']},
```

Comment:
Not sure I understand why U_BaveICT is a requirement for the magspecstitcher. Same goes for the other classes

Reply:
You are right that U_BCaveICT isn't really used for the magspecstitcher as it is, so I will remove it from the required for that analysis.  But, I like having this for the HiResMagCam analysis to get the scalars of charge and I *will* like having this for when we do UCRad2 plots of light vs charge